### PR TITLE
[DO NOT MERGE BEFORE 02/28] Make DeveloperDayBanner reusable and use for OIE Default banner

### PR DESCRIPTION
**Pull request by francoislehoux-okta**
## Description:
- **What's changed?** 
- Make DeveloperDayBanner reusable and use for OIE Default banner 

### Resolves:
* [OKTA-469571](https://oktainc.atlassian.net/browse/OKTA-469571)

[Preview](https://6218fa99898eca0943da74da--reverent-murdock-829d24.netlify.app)

***With Title, Description and CTA***
Screen Shot 2022-02-24 at 5 14 18 PM
Screen Shot 2022-02-24 at 5 14 30 PM
Screen Shot 2022-02-24 at 5 14 40 PM

***With the OIE as Default description only (what we're commiting here)***
Screen Shot 2022-02-24 at 5 44 10 PM
Screen Shot 2022-02-24 at 5 44 02 PM
Screen Shot 2022-02-24 at 5 43 54 PM

### CODE REVIEW

File: packages/@okta/vuepress-theme-prose/components/Home.vue [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
       <div class="width-adjuster">
+        <AnnouncementBanner>
+          <template v-slot:description>
+            <SmartLink :item="{ link: '/signup/' }"
+              >Developer sign ups</SmartLink
+            >
+            now utilize Okta Identity Engine beginning March 1st, 2022.
+            <SmartLink :item="{ link: '/docs/guides/oie-intro/' }"
+              >Click here</SmartLink
+            >
```
Comment (shawyu-okta) [inline packages/@okta/vuepress-theme-prose/components/Home.vue:16]: nit: formatting is a little weird here and on the previous lines too

File: packages/@okta/vuepress-theme-prose/components/Home.vue [hunk 2]
```diff
@@ -1,12 +1,23 @@
...
+            to learn more about how this impacts our developer experience &
```
Comment (shawyu-okta) [inline packages/@okta/vuepress-theme-prose/components/Home.vue:17]: see if we should use `&amp` instead - GH is also calling out the & here

File: packages/@okta/vuepress-theme-prose/global-components/AnnouncementBanner.vue [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      <div class="announcement-banner--description-wrapper">
+        <p class="announcement-banner--description">
+          <slot name="description"></slot>
+        </p>
+        <a
+          v-if="ctaHref && ctaText"
+          :href="ctaHref"
+          class="announcement-banner--cta"
+        >
+          {{ ctaText }} &rsaquo;
```
Comment (shawyu-okta) [inline packages/@okta/vuepress-theme-prose/global-components/AnnouncementBanner.vue:19]: Should the right arrow always be included? Would it be better to just leave it to the `ctaText` to specify this?

**Review by shawyu-okta — APPROVED** [review]

File: packages/@okta/vuepress-theme-prose/global-components/AnnouncementBanner.vue (hunk 3 above)
Comment (francoislehoux-okta) [reply packages/@okta/vuepress-theme-prose/global-components/AnnouncementBanner.vue:19]: That particular button always has it in the mocks but I don't mind moving it out.

File: packages/@okta/vuepress-theme-prose/components/Home.vue (hunk 1 above)
Comment (francoislehoux-okta) [reply packages/@okta/vuepress-theme-prose/components/Home.vue:16]: I'll try to run the formatter again, so many oddities in here ^^

**Review by albertchen-okta — APPROVED** [review]
🚢

**Pull request merged**